src/Service.Tests/dab-config.PostgreSql.json

```diff
@@ -2697,7 +2728,9 @@
           ],
           "target.fields": [
             "accountkey"
-          ]
+          ],
+          "linking.source.fields": [],
```

Comment:
if we are writing linking.source.fields even though they are empty, it signals a bug in CLI. Can you please create an issue tracking this?